src/ArrayToXml.php

```diff
@@ -67,8 +67,13 @@ public static function convert(
         return $converter->toXml();
     }
 
-    public function toXml(): string
+    public function toXml(bool $isPretty = false): string
```

Comment:
I like the functionality, but I'd rather have this a separate method called `prettify` that set some instance property so `toXml` will prettify the output.

Reply:
Ok, I can do that instead